app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :configure_sign_up_params, only: [:create]
+
+  def create
+    resource = User::Create.call(user: sign_up_params)
+    yield resource if block_given?
+    if resource.persisted?
+      handle_login(resource)
+    else
+      clean_up_passwords resource
+      set_minimum_password_length
+      respond_with resource
+    end
+  end
+
+  protected
+
+  def handle_login(resource)
+    if resource.active_for_authentication?
+      set_flash_message! :notice, :signed_up
+      sign_up(resource_name, resource)
+      respond_with resource, location: after_sign_up_path_for(resource)
+    else
+      set_flash_message! :notice, :"signed_up_but_#{resource.inactive_message}"
+      expire_data_after_sign_in!
+      respond_with resource, location: after_inactive_sign_up_path_for(resource)
+    end
+  end
+
+  def configure_sign_up_params
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:attribute])
```

Comment:
Same here - According to Ragnarson's Ruby style guide, I would change [:attribute] to %i(attribute).